client/components/likes/likes.tsx

```diff
@@ -1,6 +1,5 @@
 import { useEffect, useState } from 'react';
-import { LIKES_INCREMENT_STEP, LIKES_INCREMENT_DELAY } from '../../../server/src/utils/constants';
-import { LikesDisplay } from './likesDisplay';
+import * as Client_constants from '../..//utils/constants';
```

Comment:
extra slash?

Reply:
also (one small picky thing), let's keep our casing consistent, we'll stick to camel/pascal case for any imports/variables,
so `Client_constants` -> `ClientConstants`.

the casing is a little funky, but usually imports are with pascal case, variables are camel case and constants are in all caps with underscores.

Reply:
Thanks Natalie! Fixed!